app/data/AtomListStore.scala

```diff
@@ -80,7 +80,16 @@ class CapiBackedAtomListStore(capi: CapiAccess) extends AtomListStore {
         (asset \ "version").as[Long]
       }
 
-      Some(MediaAtomSummary(id, title, posterImage, contentChangeDetails))
+      val currentAsset = (atom \ "assets").as[JsArray].value.find { asset =>
+        val assetVersion = (asset \ "version").as[Long]
+        activeVersion.contains(assetVersion)
+      }
+
+      val mediaPlatform = currentAsset.flatMap { asset =>
+        (asset \ "platform").asOpt[String].map(_.toLowerCase)
+      }
```

Comment:
When we search for atoms by platform on CAPI, we check against all the assets. So if one asset in an atom has a platform of `url`, it will get returned by a query looking for `media-platform=url`.

Here instead we are checking for the latest asset. We could check all assets, in which case we could potentially show _both_ Youtube _and_ Loop if one asset is self-hosted, and the other is Youtube. This might be more confusing - or less?

Very few production atoms will contain both a Youtube and Loop asset.

Reply:
I think it makes more sense to show both Youtube and Loop if one asset is self-hosted and another is Youtube, but I can see how this can potentially be confusing either way.

Reply:
Okay, I've changed the logic so that it shows all the platforms - if the asset is inactive it shows with lower opacity, and the active asset is always the one on the right hand side.